homeassistant/components/lifx/light.py

```diff
@@ -151,15 +153,11 @@
 
 def aiolifx():
     """Return the aiolifx module."""
-    import aiolifx as aiolifx_module
-
     return aiolifx_module
 
 
 def aiolifx_effects():
```

Comment:
I know this is not the purpose of this issue, but this function is unnecessary, could it be removed?

Reply:
See above.